hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/NetworkTransactionGetReceiptHandler.java

```diff
@@ -55,16 +69,82 @@ public Response createEmptyResponse(@NonNull final ResponseHeader header) {
         return Response.newBuilder().transactionGetReceipt(response).build();
     }
 
+    @Override
+    public boolean requiresNodePayment(@NonNull ResponseType responseType) {
+        return responseType == ANSWER_ONLY || responseType == ANSWER_STATE_PROOF;
```

Comment:
We're a little inconsistent in this class regarding order of operands.  We should probably decide to either put the enum value first or second and stick to that across the class, rather than sometimes first and sometimes second.
i.e.
either match below and use
`ANSWER_ONLY == responseType || ANSWER_STATE_PROOF == responseType;`
or change the other method to match this
`responseType == COST_ANSWER;`

Reply:
I fixed get account details and receipt thank you